modules/installation-creating-azure-worker.adoc

```diff
@@ -33,13 +33,14 @@ In this example, you manually launch one instance by using the Azure Resource
 Manager (ARM) template. Additional instances can be launched by including
 additional resources of type `06_workers.json` in the file.
 
+ifdef::azure[]
 [NOTE]
 ====
-If you do not use the provided ARM template to create your worker machines, you
-must review the provided information and manually create the infrastructure. If
-your cluster does not initialize correctly, you might have to contact Red Hat
-support with your installation logs.
+By default, Microsoft {cp} places control plane machines and compute machines in a pre-set availability zone. You can manually set an availability zone for a compute node or control plane node. To do this, modify a vendor's Azure Resource Manager (ARM) template by specifying each of your availability zones in the `zones` parameter of the virtual machine resource.
```

Comment:
I see ARM acronym is already introduced  in Line 33. We can directly mention ARM template. Just a suggestion, up to you.

Reply:
Yes, good point. The first instance is a shared paragraph among install docs. Updated.